R/getResamplingIndices.R

```diff
@@ -45,17 +44,17 @@ getResamplingIndices = function(object, inner = FALSE) {
 
     # now translate the inner inds back to the outer inds so we have the correct indices https://github.com/mlr-org/mlr/issues/2409
 
-    inner_inds_translated = map(1:length(inner_inds), function(z) # map over number of outer folds
+    inner_inds_translated = lapply(1:length(inner_inds), function(z) # map over number of outer folds
 
-      set_names(
-        map(c("train.inds", "test.inds"), function(u) # map over train/test level
+      setNames(
+        lapply(c("train.inds", "test.inds"), function(u) # map over train/test level
 
           # list() -> create list for "train.inds" and "test.inds"
-          # flatten() -> reduce by one level
+          # unlist() -> reduce by one level
           # set_names(c("train.inds", "test.inds")) -> now set list names
-          flatten(
-            map(inner_inds[[z]][[u]], ~ # map over number of inner folds
-                  list(outer_inds[["train.inds"]][[z]][.x]) # the inner test.inds are a subset of the outer train.inds! That's why "train.inds" is hardcoded here
+          unlist(
```

Comment:
probably should use `recursive = FALSE`, but as I say below I dont really understand what you are trying to do here.